src/attr.c

```diff
@@ -274,8 +277,9 @@ int store_geo_attr(struct ticket_config *tk, const char *name, char *val, int no
 	if (!notime)
 		get_time(&a->update_ts);
 
+	assert(strnlen(name, BOOTH_NAME_LEN) < BOOTH_NAME_LEN);
```

Comment:
I'd rather have booth continue to run, even with a truncated attribute name. iirc, this was not unpremeditated.